### PR TITLE
📝 `Contributors`: Revise `CONTRIBUTING.md`

### PR DESCRIPTION
**Pull request by zspencer**
- https://github.com/zinc-collective/convene/issues/1520

I'm trying to streamline things back down now that we have a better idea of how things are going.

Still not entirely sure how much should live in-repo vs in-issue but I figure we'll get there as we go.

Looking for feedback re:

- Is there anything that feels *important* to keep that was deleted?
- Is there anything that has been added that feels extraneous?

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,55 +1,48 @@
...
+### 1.2 Users and Stakeholders.
+Stakeholders for `Convene` fall into one or more of the following categories:
 
-The root directory provides the human and programmatic interface for Convene
-as well as data-persistence; with two additional submodules.
+- [`Clients`], who are the legal owners of the data of their [`Spaces`] in a [`Neighborhood`].
+- [`Members`], have broad authority over [`Spaces`] and their [`Furniture`]. They are also responsible for [`Visitor`: Support], [`Visitor`: Trust & Safety], [`Member`: Support], and [`Member`: Trust & Safety].
+- [`Visitors`], who access [`Spaces`] and their [`Furniture`] according to [`Member`] specification.
+- [`Operators`], who steward the [`Neighborhood`: Infrastructure], [`Neighborhood`: Trust & Safety], and [`Client`: Support] responsibilities
+- [`Contributors`] and [`Maintainers`], who design and build [`Furniture`].
```
Comment (daltonrpruitt) [inline CONTRIBUTING.md:30]: So, are these that don't link to anything yet meant to be linked later?

Comment (zspencer) [reply CONTRIBUTING.md:30]: I am also happy to kick this back to drafts and ensure all `[` have links before merging.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,55 +1,48 @@
...
-Each issue labeled `epic` is pretty big. We tend to use checkboxes to decompose
-big issues into smaller pieces of work for people to bite off. We don't expect
-contributors to complete all the work before submitting their first patch.
-Instead, we prefer Contributors to take little bites and submit small patches so
-that we can develop tight, asynchronous feedback loops.
-
-We use labels like `design`, `documentation`, `test`, `code`, `infrastructure`,
-and `security` to indicate what kind of work is remaining on the issue.
+New contributors may want to review
+[the Zinc contribution guide](https://www.zinc.coop/contributing/) to ground themselves in the norms we try to follow across projects.
```
Comment (daltonrpruitt) [inline CONTRIBUTING.md:14]: I would argue the information in the contribution guide is not as specific as that which is being removed here. \
That said, both are meant to be somewhat general, and the reality is probably a bit more fluid than this doc could provide?

File: CONTRIBUTING.md (hunk 1 above)
Comment (daltonrpruitt) [reply CONTRIBUTING.md:30]: I am of the mindset that as long as that change is understood to exist, then this should be good to go in its present state.

File: CONTRIBUTING.md (hunk 2 above)
Comment (zspencer) [reply CONTRIBUTING.md:14]: I think there's probably some stuff worth keeping that I got rid of (for example the "We don't expect contributors to complete everything for every patch; instead we encourage people to submit pull requests early and ask for guidance and help when they need it."

**Review by daltonrpruitt — APPROVED** [review]

**Pull request merged**